### ISSUE TITLE
[Token] Raw dimension tokens update (Oct, 7)

### ISSUE THREAD

**Issue by julien-deramond** [Orange-OpenSource/ouds-ios#169]
### Description

Dimension raw tokens have been updated on October, 7.

⚠ As usual, the files generated by the parser are not perfect, so please adapt to your current tokens files.

Modification in `build/ios/raw/DimensionRawTokens+Values.swift`:

```diff
    public static let dimension1200 = dimensionBase * 24 // 96
+    public static let dimension125 = dimensionBase * 2.5 // 10
    public static let dimension1400 = dimensionBase * 28 // 112
```

```diff
    public static let dimension800 = dimensionBase * 16 // 64
+    public static let dimension850 = dimensionBase * 17 // 68
    public static let dimension900 = dimensionBase * 18 // 72
```

**Comment by pylapp** [comment]
📣 **New TestFlight ALPHA upload available** 🚀

Please, get the new build from *TestFlight* app, or contact the project maintainers if you don't have access.

- Display name: **OUDS Showcase ALPHA (155, 156, 169, 170, 172, 173, 174)**
- Version: **0.4.0**
- Build number: **1728313283**
- Build tag: **1522281**
- Build type: **ALPHA**
- Build details (GitHub): **155, 156, 169, 170, 172, 173, 174**


cc @B3nz01d @pylapp @ludovic35 @Tayebsed93 @mccart77

_This is an automated message sent with love and *Fastlane* from our CI/CD pipeline 🤘_

**Comment by pylapp** [comment]
ℹ️ Released with version [v0.4.0](https://github.com/Orange-OpenSource/ouds-ios/releases/tag/0.4.0) 🚀
